src/elements/emby-button/emby-button.css

```diff
@@ -8,20 +8,20 @@
     font-size: inherit;
     font-family: inherit;
     color: inherit;
+
+    /* These are getting an outline in opera tv browsers, which run chrome 30 */
+    outline: none !important;
     outline-width: 0;
     user-select: none;
     cursor: pointer;
     z-index: 0;
     padding: 0.9em 1em;
     vertical-align: middle;
     border: 0;
-    vertical-align: middle;
     border-radius: 0.2em;
-    /* These are getting an outline in opera tv browsers, which run chrome 30 */
-    outline: none !important;
-    position: relative;
```

Comment:
why is this gone?

Reply:
Duplicate properties and the outline should be before the more specific outline property above.

Reply:
I was referring to `position: relative;` - why did you remove that? I don't see it being duplicated, but maybe I looked in the wrong place?..

Reply:
Oh, right, sorry :)